### DISCUSSION TITLE
I can't use Lua function to open Neotree for all tabs

### DISCUSSION THREAD

**Discussion by 0xnofoot** [nvim-neo-tree/neo-tree.nvim#1293] [Q&A]
local function openNeotree()
				local tabs = vim.fn.gettabinfo()

				for _, tab in ipairs(tabs) do
					vim.api.nvim_command("tabnext " .. tab.tabnr)
					vim.api.nvim_command("Neotree show")
				end
			end

it does not work, just will open neotree in first tab.

**Comment by cseickel** [comment]
It seems that when you call the neotree command, the neovim APIs are still reporting that the first tab is focused even though you have issued the vimscript command to change tabs. Have you try using the lua api instead?

If what you want is to have the tree open in all tabs, it might be a better option to just run that command with an autocmd that runs after you switch tabs.
